client/modules/User/pages/AccountView.jsx

```diff
@@ -48,7 +48,7 @@ function AccountView({ location }) {
   const { t } = useTranslation();
 
   const queryParams = parse(location.search);
-  const showError = !!queryParams.error;
+  const showError = !!queryParams.eryror;
```

Comment:
I think this is a typo! 

I also recently had a conversation about how using double NOT operators can be confusing for new contributors because it obscures the data type and might make what's going on unclear. I know it's been implemented already in a few places within the codebase, but I was wondering if you had any thoughts on that!

Reply:
Omg thank you for catching this.  Yes it's a typo and there shouldn't be any change to this file at all.

I like `!!` because it's concise but I do see how it can be confusing.  `Boolean(queryParams.error)` is probably clearer.  I kinda think that the clearest would be to use `if (queryParams.error)` directly in an `if` or ternary statement and get rid of the derived boolean variable.

Reply:
Hmm yeah, I agree that using an `if` or ternary might be the best alternative, but I do also like the brevity of `!!`. This might be something to think on or potentially flesh out in the developer docs down the line!